### ISSUE TITLE
Clearer documentation for `Bio.SwissProt.parse` function 

### ISSUE THREAD

**Issue by kamurani** [biopython/biopython#4294]
It would be great if there could be some clearer use cases / examples / explanations of what is expected input for the `parse()` function in `Bio.SwissProt`.  

From taking a look, `read` parses a single entry of a SwissProt-formatted text file (such as one you'd find from downloading from uniprot, e.g. https://rest.uniprot.org/uniprotkb/O96017.txt ).

However, in the `parse` documentation they use an example of a similar text entry: 

```python
>>> from Bio import SwissProt
>>> example_filename = "SwissProt/P68308.txt"
>>> with open(example_filename) as handle:
...     records = SwissProt.parse(handle)
```
I'm not really sure what's going on here, as presumably there is only one entry in this downloaded text file? And therefore not sure why you would use `parse` to get a generator for multiple `Record` objects? (I also couldn't find any documentation on the Uniprot site regarding downloading multiple `.txt` entries in a *single* file -- the only way I found was doing a query that returned multiple entries.  I would like to be able to simply provide a list of IDs and retrieve all the SwissProt knowledge data as a single file, if anyone is aware of how to do this). 

Additionally, as another part of my use case, would it be a good for a `write` function for a `Bio.SwissProt.Record` object? 

As an example, one might:
- retrieve from UniProt API a bunch of `Record` objects 
- maybe this was from a useful query (i.e. can't reproduce just by simply providing Uniprot IDs) 
- want to store this easily and re-read later from disk (using `parse`). 

Thanks in advance for some clarity regarding this and any feedback.  

TLDR: 
- not sure what the `"SwissProt/P68308.txt"` should look like in the documentation's example if using `parse`
- not sure how to retrieve multiple entries appropriate for `parse` using uniprot and a list of IDs 
- possibly new feature for `write` function that writes SwissProt `Record` objects to disk?

**Comment by peterjc** [comment]
The SwissProt format lives on as the text version of UniProt files, but traditionally their database releases (like NCBI GenBank) came as large multi-record text files. They still do, e.g. https://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/complete/uniprot_sprot.fasta.gz - loads of individual record concatenated together.

I see very little reason to want a ``Bio.SwissProt.Record`` writer, but perhaps you have a use case? It is generally safer to save and work from the original file as given.

There is a very minimal stub of the former SwissProt API available in the ``get_sprot_raw`` function under ``Bio.ExPASy``. Something up to date for UniProt might be useful though.

**Comment by kamurani** [comment]
Thanks Peter.  I've actually been using the API wrapper for getting the records rather than parsing a bulk download file (as far as I know this is the easiest way to get records for a subset of uniprot IDs).  Other than that you could retrieve the inidividual `.txt` entries and download, and parse one file at a time with `read`. 

```python
# For each uniprot_id in a subset ...
handle = ExPASy.get_sprot_raw(uniprot_id)
record = sp.read(handle)
``` 

Once I have this list of records in memory, I was hoping to simply dump them into a file so that I can read from this on the second run.  Basically I want to continue using `ExPASy.get_sprot_raw` for any new uniprot IDs that might come up; but read from a file if they've already been saved.  And I am trying to avoid simply downloading the entire database file. 

Hopefully that makes sense.  Let me know if this is a viable use case? 

Cheers

**Comment by mdehoon** [comment]
@kamurani  

You could do the following:
```python
handle = ExPASy.get_sprot_raw(uniprot_id)
output = open("myfile.txt", 'w')
data = handle.read()
output.write(data)
output.close()
handle = open("myfile.txt")
record = sp.read(handle)
```

**Comment by peterjc** [comment]
Yes, like @mdehoon's example write the SwissProt format data from the network directly to disk.

Even if Biopython did offer to output in that format, this kind of task is rarely 100% identical to the input file (we see this with GenBank files for example). So no, I don't see this as a good use-case. If you were editing the records in some way, then perhaps.

See also the http://togows.org/ API available via Biopython, which does let you download multiple records at once. For a simple task like this, you would just be using it as a super-proxy but it can do much more. However, it means routing your traffic via Japan, and doesn't seem like the best use of their resources either.

**Comment by peterjc** [comment]
I don't see any obvious changes we can make to the ``read`` and ``parse`` function docstrings to clarify what we mean by single vs multi record plain text SwissProt files:

https://github.com/biopython/biopython/blob/master/Bio/SwissProt/__init__.py

Please reopen the issue if you can any specific suggestion. Thank you.
